.github/workflows/ci.yml

```diff
@@ -9,14 +9,18 @@ on:
 env:
   CACHE_VERSION: 1
   LATEST_Z3_VERSION: "4.8.14"
+  # Some of the runners (e.g., Windows) use very recent versions of CMake that
+  # are no longer explicitly supported by the various solvers. For now, we try
```

Comment:
I'm confused by the combination of "very recent" and "no longer"? Should it either be "very old" or perhaps "not yet"?

Reply:
I've changed this to:

https://github.com/GaloisInc/what4-solvers/blob/c6288b725bc7765c48c509591835480f77450007/.github/workflows/ci.yml#L12-L15

Better?